lib/webpack.config.js

```diff
@@ -381,8 +389,17 @@ const test = (base, { main }) =>
     },
   })
 
-const prod = (base, { main }) =>
-  merge.smart(base, {
+const prod = (base, { main, packageJson }) => {
+  let template,
+    templatePath = getTemplatePath(packageJson)
+  if (templatePath !== undefined) {
+    template = resolve(templatePath)
+  } else {
+    template = fs.existsSync(resolve('src/main/webapp/index.html'))
+      ? resolve('src/main/webapp/index.html')
+      : resolve('src/main/webapp/index.js')
+  }
```

Comment:
```js
let template,
    templatePath = getTemplatePath(packageJson)
  if (templatePath !== undefined) {
    template = resolve(templatePath)
  } else {
    template = fs.existsSync(resolve('src/main/webapp/index.html'))
      ? resolve('src/main/webapp/index.html')
      : resolve('src/main/webapp/index.js')
  }
```

This code could be pulled out into a method as it's used twice (once in prod and once in dev)